INSTALL.md

```diff
@@ -45,7 +46,7 @@ You can also use their abbreviations:
 sudo ./scripts/install.sh -r <REGION> -s <STAGE>
 ```
 
-#### Troubleshooting
+#### Linux or OSX Troubleshooting

```

Comment:
Nitpick: I think OSX is just called macOS now.
https://en.wikipedia.org/wiki/OSX_(disambiguation)
`OS X was until September 2016 the name of an operating system family for Apple computers, thereafter known as macOS. `

Reply:
a little TIL!